src/models/legend.js

```diff
@@ -252,7 +252,13 @@ nv.models.legend = function() {
                     xpos;
                 series
                     .attr('transform', function(d, i) {
-                        var length = d3.select(this).select('text').node().getComputedTextLength() + padding;
+                        var length = 0
+                        try {
+                            length = d3.select(this).select('text').node().getComputedTextLength() + padding;
+                        }
+                        catch(e) {
+                            length = nv.utils.calcApproxTextWidth(d3.select(this).select('text')) + padding;
+                        }
```

Comment:
Just a preference but I would avoid the exception and instead do something like:
```
var textNode = d3.select(this).select('text');

length = (textNode.getComputedTextLength ? textNode.getComputedTextLength() : nv.utils.calcApproxTextWidth(textNode)) + padding;
```